.golangci.yml

```diff
@@ -47,6 +62,32 @@ linters:
     - whitespace
 
 linters-settings:
+  # Addig the following section based on https://github.com/golangci/golangci-lint/issues/3877
+  depguard:
```

Comment:
i would drop this, this is a bit expensive to maintain and not sure what values this adds

for example we dropped this check in the sigstore org

Reply:
@cpanato  I added this because not having this was breaking my local run of `make verify`  :( Should I disable that plugin entirely instead ?

Reply:
yes, remove that check